kube/services/jupyterhub/jupyterhub_config.py

```diff
@@ -94,6 +116,12 @@
         }
     }
 }
+# TODO: Remove this below line. But need this here for now for integration test purposes.
```

Comment:
Did you mean to remove some of this?

Reply:
Nahh, I'm just submitting for review prematurely. Will remove that later